examples/online-shop/README.md

```diff
@@ -0,0 +1,61 @@
+# Ballerina OracleDB module Example use case
+
+## Overview 
+This example demonstrates how to use the ballerina oracledb module to execute, query and call etc. in an oracle database. 
+
```

Comment:
Shall we write an overview of the use case(scenario) we implemented here

Reply:
Updated in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/36/commits/726dbeffdcc1c1f0c1df33b9bd7e62f715f4afe5